doc/quickstart.rst

```diff
@@ -81,14 +81,22 @@ This method will load the complete trajectory into memory and make positions, or
 .. code-block:: python
 
     traj.load_arrays()
-    traj.N              # M  -- frame sizes
-    traj.positions      # MxNx3 array
-    traj.orientations   # MxNx4 array
-    traj.types          # MxN array
-    traj.type_ids       # MxN array
-    traj.type           # list of type names ordered by type_id
-
-    # where M=len(traj), N=max((len(f) for f in traj))
+    traj.N               # M
+    traj.positions       # MxNx3
+    traj.orientations    # MxNx4
+    traj.velocities      # MxNx3
+    traj.mass            # MxN
+    traj.charge          # MxN
+    traj.diameter        # MxN
+    traj.moment_inertia  # MxNx3
+    traj.angmom          # MxNx4
+    traj.image           # MxNx4
+    traj.types           # MxN
+    traj.type_ids        # MxN array
+    traj.type            # list of type names ordered by type_id
+
+    # where M=len(traj) is the number of frames and
+    # N=max((len(f) for f in traj)) is the number of particles
```

Comment:
is the maximum number of particles in any frame.

Also we should add a test for accessing data from grand-canonical simulations / changing number of particles. I'm not sure what our trajectory attributes really do right now in that case.

Reply:
I'll open an issue for that and do it at some point. Thanks